### ISSUE TITLE
Working together with the alerts preprocessor

### ISSUE THREAD

**Issue by szabgab** [Michael-F-Bryan/mdbook-linkcheck#93]
Firs of all thanks for making this plugin!

The [alerts preprocessor](https://crates.io/crates/mdbook-alerts) let's me use code like this:

```
> [!NOTE]
> Useful information that users should know, even when skimming content.
```

but the linkcheck does not like it and gives me this warning:

```
warning: Potential incomplete link
   ┌─ preprocessors/alerts.md:45:3
   │
45 │ > [!NOTE]
   │   ^^^^^^^ Did you forget to define a URL for `!NOTE`?
   │
   = hint: declare the link's URL. For example: `[!NOTE]: http://example.com/`
```

How could I make the linkcheck accept this and ignore these entries?

**Comment by mightyiam** [comment]
Something like this seems to do the trick:

```toml
[preprocessor.alerts]
renderers = ["html", "linkcheck"]
```
